packages/api/src/store/table.ts

```diff
@@ -146,17 +146,21 @@ export default class Table<T extends DBObject> {
     }
   }
 
-  async update(id: string, doc: T) {
+  async update(id: string, doc: T, where: string = '', doNotThrow = false) {
```

Comment:
I don't love this interface. What about this one instead?
```typescript
async update(
    query: string | Array<SQLStatement>,
    doc: T,
    opts: UpdateOptions = {throwIfEmpty = true},
):
```
If `query` is a string it's the same as the current function, otherwise
Advantages:
* Very similar to the interface of find(), so it's consistent
* Usage of SQLStatement protects us from SQL injection down the line when someone does a WHERE with 
* `opts` object means we can add more options later without adding more arguments

Your call above could be rewritten as:
```typescript
      const upRes = await db.stream.update(
        [
          sql`id = ${stream.id}`,
          sql`(data->>'lastSeen')::bigint = ${stream.lastSeen}`,
        ],
        { isActive: false },
        { throwIfEmpty: false }
      )

Reply:
fixed